protocol/auth.go

```diff
@@ -26,6 +26,9 @@ type AuthorizationResponseMessage struct {
 
 	From string `json:"from,omitempty"`
 	To   string `json:"to,omitempty"`
+
+	CreatedTime *int64 `json:"created_time,omitempty"`
+	ExpiresTime *int64 `json:"expires_time,omitempty"`
```

Comment:
should we use uint64? can it be negative?

Reply:
I pick uint64 because of `Unix()` functions: 
`func (t time.Time) Unix() int64`
`func time.Unix(sec int64, nsec int64) time.Time`

it returns int64 and accept int64 to create time object in go